python/ray/train/v2/tests/test_worker_group.py

```diff
@@ -536,6 +565,108 @@ def conditional_failure():
     # If more tests are added below this, they may not be able to run.
 
 
+def test_check_cluster_resources_and_raise_if_insufficient(monkeypatch):
+    """Test _check_cluster_resources_and_raise_if_insufficient method."""
+    wg = _default_inactive_worker_group()
```

Comment:
update to just test the staticmethod and remove this wg isntance

Reply:
nit: can condense this test with a `_assert_resource_check(available_resources, resources_per_worker, num_workers, should_raise)` helper